src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,17 @@
+.main {
+  display: flex;
+  justify-content: center;
+
+  &__catalog {
+    max-width: 1024px;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    padding: 0 40px;
+    margin-top: 50px;
+  }
+}
```

Comment:
move to mix (main__catalog) only those properties which are related to outer spacing and positioning and move all other styles to catalogue itself